### PR TITLE
Ria 3302 ho end appeal

### PR DESCRIPTION
**Pull request by Lee365**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3302

### Change description ###
RIA-3302_ho_end_appeal

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseNotificationsHandlerTest.java [hunk 1]
```diff
@@ -116,7 +116,8 @@ public void it_can_handle_callback() {
                         Event.REQUEST_RESPONDENT_REVIEW,
                         Event.LIST_CASE,
                         Event.EDIT_CASE_LISTING,
-                        Event.ADJOURN_HEARING_WITHOUT_DATE
+                        Event.ADJOURN_HEARING_WITHOUT_DATE,
+                        Event.END_APPEAL
```
Comment (jkumar-moj) [inline src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseNotificationsHandlerTest.java:120]: could you please update the other test case to verify triggering the home-office-api.

**Review by jkumar-moj — CHANGES_REQUESTED** [review]

**Review by jkumar-moj — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseNotificationsHandlerTest.java (hunk 1 above)
Comment (Lee365) [reply src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseNotificationsHandlerTest.java:120]: "END_APPEAL" added to other test

**Review by jkumar-moj — DISMISSED** [review]
👍

**Review by jkumar-moj — APPROVED** [review]

**Review by Srihari-Nakkala — DISMISSED** [review]
LGTM

**Review by jkumar-moj — APPROVED** [review]

**Pull request merged**